src/types/tau-prolog/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "tau-prolog/modules/core" {
+    export const type: any;
```

Comment:
It would be nice to eventually improve these typings; e.g. to add typings for all properties that you directly access. Here you could already use something like the following (or more specific, if you know more about `Session` and the parameter type / return type of `format_answer`):
```
export const type: {
  Session: any;
};
export const format_answer: (x: any) => any;
```

Reply:
```typescript
export namespace type {
  export class Session;
}
export function format_answer(x: any): string;
```

I think.